dev-lisp/sbcl/sbcl-2.1.10.22_44e0d89ff.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="A high performance Common Lisp compiler"
+DESCRIPTION="Steel Bank Common Lisp (SBCL) is a high performance Common \
+Lisp compiler. It is open source / free software, with a permissive license. \
+In addition to the compiler and runtime system for ANSI Common Lisp, it \
+provides an interactive environment including a debugger, a statistical \
+profiler, a code coverage tool, and many other extensions."
+HOMEPAGE="https://sbcl.org"
+COPYRIGHT="2002 Gerd Moellmann"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="git://github.com/sbcl/sbcl.git"
```

Comment:
could you please instead use a tarball? see for example:
https://github.com/haikuports/haikuports/blob/4f180bdb94db9749ae800820b271e262a12754aa/net-im/purple_discord/purple_discord-20210729.recipe#L9

Reply:
updated to use a tarball, thanks!